aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/TestLocalAliyunOSS.java

```diff
@@ -58,37 +58,38 @@ private static void assertThrows(Runnable runnable, String expectedErrorCode) {
         .isEqualTo(expectedErrorCode);
   }
 
-  @Before
+  @BeforeEach
   public void before() {
-    OSS_TEST_RULE.setUpBucket(bucketName);
+    OSS_TEST_EXTENSION.setUpBucket(bucketName);
   }
 
-  @After
+  @AfterEach
   public void after() {
-    OSS_TEST_RULE.tearDownBucket(bucketName);
+    OSS_TEST_EXTENSION.tearDownBucket(bucketName);
   }
 
   @Test
   public void testBuckets() {
-    Assume.assumeTrue(
-        "Aliyun integration test cannot delete existing bucket from test environment.",
-        OSS_TEST_RULE.getClass() == AliyunOSSMockRule.class);
+    Assumptions.assumeTrue(
```

Comment:
it would be better to use Assumptions from AssertJ as well: `Assumptions.assumeThat(OSS_TEST_RULE.getClass()).isEqualTo(...)`